meteor/lib/lib.ts

```diff
@@ -771,15 +782,15 @@ export const caught: <T>(v: Promise<T>) => Promise<T> = (f => p => (p.catch(f),
  * Blocks the fiber until all the Promises have resolved
  */
 export const waitForPromiseAll: <T>(ps: Array<Promise<T>>) => Array<T> = Meteor.wrapAsync(function waitForPromises<T> (ps: Array<Promise<T>>, cb: (err: any | null, result?: any) => T) {
```

Comment:
This function now freezes execution.
There is a call to Meteor.wrapAsync still and its callback is never called, so it never returns.

Perhaps this is what was intended?
```
export function waitForPromiseAll<T>(ps: Array<Promise<T>>): Array<T> {
	return waitForPromise(Promise.all(ps))
}
```